content/en/users/getting-started/openstack/_index.md

```diff
@@ -1,11 +1,32 @@
 ---
 title: "Using OpenStack Providers"
-linkTitle: "Using OpenStack Providers"
+linkTitle: "OpenStack Providers"
 type: docs
-weight: 10
-draft: true
+weight: 20
 description: >
-  How to interact with OpenStack providers in the EGI Cloud
+  How to interact with OpenStack providers in the EGI FedCloud
```

Comment:
I would replace by EGI Cloud Compute

Reply:
Why? Are all the OpenStack services cloud compute? Keystone? Cinder? Swift?

Reply:
My first problem was here to see another mention of FedCloud (that I understood is a term a bit deprecated).
Then I mentioned Cloud Compute as it's one of the real services that covers the IaaS usage. Swift would be under the Object Storage service, and I expect that related documentation may be there too.
I'm not sure EGI Cloud is a naming we are actively using/promoting/advocating.
@enolfc ?

Reply:
I am slowly updating everything to EGI FedCloud (and will amend this naming where grid/htc resources ar einvolved - although for me those are cloud as well, as I guess would be to any newcomer to EG services)

Reply:
In my view `FedCloud` usually refers to the EGI providers that have that tag in GOCDB, mostly delivering some of the OpenStack services, but also to the group of people that follows the fedcloud mailing list and connect to the meetings. I would avoid this to refer to the infrastructure as here, it does not bring much clarity.

`EGI Cloud` does not exist as a term that we officially use. I agree with @thebe14  that most services in EGI portfolio could be named cloud, but I would use EGI Services instead as we don't have EGI Cloud used anywhere and people would just get confused as cloud is traditionally seen by our federation members as anything which is not HTC

So to use validated terms, please use [EGI infrastructure](https://ims.egi.eu/display/EGIG/EGI+Infrastructure) for referring to the providers

Reply:
The documentation contains EGI Federated Cloud in a lot of places. I started using "EGI FedCloud" as a short form of that, so that I no longer use e.g. "EGI Cloud services".